### ISSUE TITLE
Store badges are out of date

### ISSUE THREAD

**Issue by chrisgarrity** [LLK/scratchjr-website#266]
We are using out of date store badges (originals were downloaded when the app was released).

Should replace with:

[badges.zip](https://github.com/LLK/scratchjr-website/files/4880831/badges.zip)

**Comment by IamTregsthedev** [comment]
Does this Still Need help?

**Comment by IamTregsthedev** [comment]
Here are the Badges, that will be needed to change.
![google-play-badge](https://user-images.githubusercontent.com/21000033/103712856-de615780-4f88-11eb-9052-88e760560487.png)

**Comment by IamTregsthedev** [comment]
> Here are the Badges, that will be needed to change.
> ![google-play-badge](https://user-images.githubusercontent.com/21000033/103712856-de615780-4f88-11eb-9052-88e760560487.png)

App store as well.

**Comment by chrisgarrity** [comment]
@IamTregsthedev yes this is still open and helpwanted. There is a zip file attached to the issue that contains the google and apple badges.
